src/table/cells/handle-cell-focus.js

```diff
@@ -41,3 +45,27 @@ export const handleResetFocus = (focusedCellCoord, rootElement, shouldRefocus, h
     rowElements[0]?.getElementsByClassName('sn-table-cell')[nextcell[1]].setAttribute('tabIndex', '0');
   }
 };
+
+export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement }) => {
+  const MIN_ROW_COUNT = 2;
+  if (!tableSection.current.scrollTo) return;
```

Comment:
isn't `tableSection.current` undefined before ref is set? In that case add a `?`

Reply:
no, it's always an empty object, but we could also check that by adding an optional chaining (?)

Reply:
✅ Fixed